src/utils/GraphhopperUtils.js

```diff
@@ -148,15 +212,31 @@ async function fetchRoutes(end: string, start: string, departureTimeDateNow: str
       true,
     ),
   ]).then((vals) => {
```

Comment:
why not just `[busRouteNowRequest, busRouteBufferedRequest, walkingRouteRequest] = await ...`
we shouldn't be using `.then()` with `Promise`